R/function_defaults.R

```diff
@@ -0,0 +1,18 @@
+
+function_defaults <- function(fun.name, out=c('package','def.funs')){
+  funs <- list('points'=
```

Comment:
Yeah, looks like we are doing different things for formals between these functions, and that should be consistent. For these functions, I take formals from both `plot.xy` and `lines.default`. In config it just uses one of those